### ISSUE TITLE
To-Dos / Issues / PRs screen

### ISSUE THREAD

**Issue by yusdacra** [git-touch/git-touch#131]
It would be handy to have a seperate screen for these. While for GitHub the notifications screen kinda works I think a dedicated screen would be much more handy (it would also work with other services too).

**Comment by shreyas1599** [comment]
@yusdacra Do you mean the list of all issues/PRs associated with a user for each service? For example, in GitHub, issues and PR tabs in the highlighted section below? 

![image](https://user-images.githubusercontent.com/28999492/105277184-ee07af80-5bc8-11eb-8e9f-ad971cdb0c0c.png)

**Comment by yusdacra** [comment]
Yes. I'm not sure if that's the best place for it though. For other services which don't have 'notifications' we should put the list of issues / PRs at the bottom bar.
